src/3d/qgscameracontroller.h

```diff
@@ -197,6 +197,13 @@ class _3D_EXPORT QgsCameraController : public QObject
      */
     bool willHandleKeyEvent( QKeyEvent *event );
 
+    /**
+     * Reacts to the shift of origin of the scene, updating camera pose and
+     * any other member variables so that the origin stays at the same position
+     * relative to other entities.
+     */
```

Comment:
This is exposed to sip, so:

```suggestion
     * relative to other entities.
     * \since QGIS 3.42
     */
```

Reply:
Added in [61ea829](https://github.com/qgis/QGIS/pull/59468/commits/61ea8295bc1a86496f8c402dc5c0503d882e45f7)